### ISSUE TITLE
Accessibility: Unlabeled and unusable elements with talkback (0.5XMR)

### ISSUE THREAD

**Issue by shortwavesurfer2009** [m2049r/xmrwallet#933]
On the wallet select screen

- [ ] the "+" icon is labeled as "create new wallet" instead of something like "create or restore wallet" which would be more accurate

- [ ] When the "+" is pressed the menu options for "create new wallet", "restore view-only wallet", "restore wallet from private keys", "restore wallet 25 word seed", and "import wallet", cannot be focused by talkback at all and cannot be used.

- [ ] When the "+" button is pressed the "x" it becomes still reads as "create new wallet" instead of "close"

On the nodes screen

- [ ] the heart icon for saving or unsaving nodes reads as "unlabeled". I suggest "remove saved node" when the heart is fully red and "save node" when it is only the outline

When a wallet is open

- [ ] the three line hamburger menu at the top left reads as "unlabeled"

Edit: [A bounty to fix this](https://bounties.monero.social/posts/122/0-100m-monerujo-more-accessibility-work)

**Comment by anhdres** [comment]
Thank you for taking the effort to help us make the wallet more accesible.

**Comment by shortwavesurfer2009** [comment]
@anhdres its selfish since I rely on accessibility myself. It just happens to be good for more than just me.

**Comment by anhdres** [comment]
I don't think it's selfish. It's like me wanting to have real ARS/XMR prices in the app, it's useful for me, but also to a lot of people that doesn't know how to ask for it.

If it wasn't for you, we'd simply not notice. Keep us pinged and we'll do our best.

**Comment by plowsof** [comment]
i have confirmed that all issues have been addressed in detherminals branch / workflow run @ https://github.com/detherminal/xmrwallet/actions/runs/11159241005
